src/scripts/clipperUI/componentBase.ts

```diff
@@ -12,6 +12,7 @@ export interface EnableInvokeParams {
 export interface EnableAriaParams extends EnableInvokeParams {
 	ariaSetName: string;
 	ariaSetDirection: AriaNavDirection;
+	autoSelect: boolean;
```

Comment:
It might be good to make this an optional param since this will be false most of the time